test/features/formItemExtra.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import faker from 'faker';
+
+import { Tester } from '@mighty-justice/tester';
+
+import { COMPONENT_GENERATORS } from '../factories';
+import { IValue } from '../../src/props';
+
+const SUPPORTING_COMPONENTS = [
+  'Form',
+  'FormCard',
+  'FormDrawer',
+  'FormModal',
+];
+
+function testFormat (value: IValue) {
+    if (value) {
+      return <span id="test">Hello</span>
+    }
+    return null;
+}
+
+// Tests that users can pass in optional extraFieldConfig props to render an extra under form item.
+describe('formItemExtra', () => {
+  SUPPORTING_COMPONENTS.forEach(componentName => {
+    it(`Form item extras correctly renders in ${componentName}`, async () => {
+      const { ComponentClass, propsFactory } = COMPONENT_GENERATORS[componentName]
+        , name = faker.lorem.sentence()
+        , props = propsFactory.build({
+            fieldSets: [[{ field: 'name'
```

Comment:
Formatting for the brackets of `fieldSets` is a little off. I would put attributes on their own lines without brackets

Reply:
I just checked, elsewhere in the codebase it's all on one line. Should I follow that or separate onto new lines?

Reply:
Either way, just not halfway!